File: ObjcSample/ObjcSample/ViewController.m
//
//  ViewController.m
//  ObjcSample
//
//  Created by Lina Choi on 2020/12/01.
//  Copyright © 2020 anil. All rights reserved.
//

#import "ViewController.h"
#import "ObjcSample-Swift.h"
#import "ObjcSample-Bridging-Header.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.

    MyClass* f = [[MyClass alloc] init];
    [f introduce];
//    [f introduceBy: "aa" with: 1]
    MyClass2* ff = [[MyClass2 alloc] init];
//    [ff ]
    [ff class2];
    
}


@end

File: AnimationSample/AnimationSample/ViewController.swift
//
//  ViewController.swift
//  AnimationSample
//
//  Created by Lina Choi on 2020/11/19.
//  Copyright © 2020 anil. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func topDownAnim(_ sender: Any) {
        let vcName = self.storyboard?.instantiateViewController(withIdentifier: "TopDownViewController")
        self.present(vcName!, animated: true, completion: nil)
    }
    
}

